gtest-parallel

```diff
@@ -127,17 +128,26 @@ class FilterFormat:
 
   tests = {}
   outputs = {}
-  started = []
-  finished = []
   failures = []
+  finished = []
+
+
+  def write_logs_to_file(self, bundle_name, tests_ids):
+    with open(os.path.join(self.logs_dir, bundle_name), 'w') as logs_bundle:
+      for test_id in tests_ids:
+        log_file = self.outputs[test_id]
+        test_name = os.path.basename(log_file)
+        logs_bundle.write('TEST: ' + test_name + '\n\n')
```

Comment:
Do we need this extra information? I'd prefer if the log looks exactly like a regular gtest output, like the test would have been executed without gtest-parallel.